OrbitGl/App.cpp

```diff
@@ -168,6 +168,23 @@ void OrbitApp::OnCaptureComplete() {
       });
 }
 
+void OrbitApp::OnCaptureCanceled() {
+  main_thread_executor_->Schedule([this]() mutable {
+    if (open_capture_failed_callback_) {
+      open_capture_failed_callback_();
+    }
+    ClearCapture();
+  });
+}
+
+void OrbitApp::OnCaptureFailed(std::string error_message) {
+  if (open_capture_failed_callback_) {
+    open_capture_failed_callback_();
+  }
```

Comment:
Should this also be called on the main thread?

Reply:
There is no need to. But if you think it is more clear, then I can move it to the main thread.

Reply:
I moved this on the main thread, as I am now also clearing the capture on failure.